sharding/client.go

```diff
@@ -120,12 +120,22 @@ func (c *Client) unlockAccount(account accounts.Account) error {
 	pass := ""
 
 	if c.ctx.GlobalIsSet(utils.PasswordFileFlag.Name) {
-		blob, err := ioutil.ReadFile(c.ctx.GlobalString(utils.PasswordFileFlag.Name))
+		file, err := os.Open(c.ctx.GlobalString(utils.PasswordFileFlag.Name))
 		if err != nil {
-			return fmt.Errorf("unable to read account password contents in file %s. %v", utils.PasswordFileFlag.Value, err)
+			return fmt.Errorf("unable to open file containing account password %s. %v", utils.PasswordFileFlag.Value, err)
 		}
-		// TODO: Use bufio.Scanner or other reader that doesn't include a trailing newline character.
-		pass = strings.Trim(string(blob), "\n") // Some text files end in new line, remove with strings.Trim.
+		scanner := bufio.NewScanner(file)
+		scanner.Split(bufio.ScanWords)
+		passwordSuccess := scanner.Scan()
+		if !passwordSuccess {
```

Comment:
No need for this one time variable, please use 
```
if !scanner.Scan() {
 ...
}
```

Reply:
Alright, made the changes and removed capitalization of error